includes/application/forwardApplication.php

```diff
@@ -63,26 +63,55 @@
 	
 	if( needApprover($status) )
 	{
-		echo <<<HTML
+		if( isset($_SESSION['PostName']) )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->